client/app/components/dialog.vue

```diff
@@ -0,0 +1,53 @@
+<template lang="pug">
+    div(v-bind:aria-hidden="hidden", role="dialog")
+        div(role='separator', v-on:click="onClose")
+        .wrapper
+            div(role='contentinfo')
```

Comment:
If I remove this div, it breaks rules defined into `cozy-ui` what should I do? https://github.com/cozy/cozy-ui/blob/master/src/stylus/cozy-ui/components/dialog.styl#L20